src/ios/CTKAdSettingsManager.m

```diff
@@ -28,8 +28,8 @@ @implementation CTKAdSettingsManager
 
 RCT_EXPORT_MODULE()
 
-RCT_EXPORT_METHOD(addTestDevice:(NSString *)deviceHash) {
-  [FBAdSettings addTestDevice:deviceHash];
+RCT_EXPORT_METHOD(addTestDevice) {
+  [FBAdSettings addTestDevice:[FBAdSettings testDeviceHash]];
```

Comment:
Can we use `constantsToExport` to export `deviceHash` and make `addTestDevice` accept any string, rather than implicitly using own device hash when argument is undefined?